armor-write/src/main/java/com/rapid7/armor/store/S3WriteStore.java

```diff
@@ -323,44 +324,30 @@ public void copyShard(ShardId shardIdDst, ShardId shardIdSrc) {
       return;
     }
 
+    String currentShardKey = null;
     try {
+      List<S3ObjectSummary> shardObjects = getShardObjects(shardIdSrc);
```

Comment:
Rename getShardObject to getCurrentShardObjects to denote we are gettting all the objects from the current transaction only.